plugin/src/main/kotlin/com/automattic/android/PublishToS3Plugin.kt

```diff
@@ -1,37 +1,48 @@
 package com.automattic.android.publish
```

Comment:
Minor (🔍): FYI that the package declaration does not match the actual file location. PS: This is minor and unrelated to this change, but thought to mention it anyway, just in case, feel free to ignore.

Note: Making this change would mean that the package would need to be updated everywhere, tests included. And, I am not sure if having this `com.automattic.android.publish` instead of `com.automattic.android` is intended here, so please do take this advise with a grain of salt. It is just that imports would be significantly improved if that change were to be applied... 🤷

Reply:
I noticed this as I was working on this PR and hoped that I could open a PR to address it before anyone else notices. 😅 

I think it'd be better to put the correct folder structure in place rather than changing the `package`, which would make it harder to review the PR if it's combined with other changes. That's why I was planning to do this in a separate PR.

I'll try to take care of that today, but if I can't, we can do this anytime as it shouldn't have any impact on how the plugin works.

Reply:
Great, thanks @oguzkocer and yes we can do this anytime! 💯